### ISSUE TITLE
Add "gofmt -s" to "make fmt" target in Makefile

### ISSUE THREAD

**Issue by LalatenduMohanty** [minishift/minishift#451]
Currently `fmtcheck` target uses `-s` which simplifies the code on top of formatting.  However "fmt" does not uses it. So some of the CI jobs fails as "make fmtcheck" complains where "make fmt"(which developers perform before sending the pull request)

```
.PHONY: fmt
fmt:
        @gofmt -l -w $(SOURCE_DIRS)

.PHONY: fmtcheck
fmtcheck:
        @gofmt -l -s $(SOURCE_DIRS) | read; if [ $$? == 0 ]; then echo "gofmt check failed for:"; gofmt -l -s $(SOURCE_DIRS); exit 1; fi
```

**Comment by LalatenduMohanty** [comment]
This is fixed as part of #443

**Comment by hferentschik** [comment]
This has already been addressed :-)
